### PR TITLE
Add invalid query behavior

### PR DESCRIPTION
**Pull request by aswanson-nr**
## Summary

[NOTE]: # ( Provide a brief overview of the changes. )
* Added an ApplicationContainer to hold values that are used across the
application
* Updated endpoints to use ApplicationContainer
* Added INVALID-QUERY behavior
* Added UAT's to cover the new behavior

## Checklist
[NOTE]: # ( Just check the ones applicable to this pull request. )
- [x] Documentation updated
- [x] Unit tests updated
- [x] User Acceptance Tests updated

## Deployment Configuration for Testing
[NOTE]: # ( Provide a deployment configuration to use during manual testing, if applicable. )

## Related Issues
[NOTE]: # ( Provide links to any related Github issues. )

## Related Pull Requests
[NOTE]: # ( Provide links to any related pull requests. )

## Reviewers
[NOTE]: # ( Mention reviewers here! )

### CODE REVIEW

File: javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import com.newrelic.lib.ApplicationContainer;
+import com.newrelic.lib.Inventory.MySQLRepository;
+
+public class InvalidQueryBehavior extends Behavior
+{
+    public static final String Name = "INVALID-QUERY";
+
+    public InvalidQueryBehavior()
+    {
+        this.setName(InvalidQueryBehavior.Name);
```
Comment (moonlight-komorebi) [inline javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java:15]: are both lines 10 & 14 needed? 10 in particular. it looks like the same value goes to two places which are effectively doing the same thing -- get/set the name.

File: javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public void Execute() throws SQLException, Exception
+    {
+        super.Execute();
+        var appConfig = ApplicationContainer.getInstance().GetAppConfigRepository();
+        var mySQLConfig = appConfig.FindMySQLConfiguration();
+        if (mySQLConfig != null)
+        {
+            var repo = MySQLRepository.getInstance(mySQLConfig);
```
Comment (moonlight-komorebi) [inline javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java]: can this be injected into the class? 🙈 
if not, or the lift seems too difficult, feel free to ignore this.

Comment (aswanson-nr) [reply javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java]: Absolutely! I was still on my weird logic from Nodetron, updating now...

File: javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java (hunk 1 above)
Comment (aswanson-nr) [reply javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java:15]: You know, I'm not actually sure about this one. It's the same model that the other behaviors follow. However, the private value appears to only be used during logging.

**Review by moonlight-komorebi — APPROVED** [review]

**Pull request merged**